.circleci/config.yml

```diff
@@ -149,7 +149,7 @@ jobs:
       - attach_workspace:
           at: /tmp/
       # check that no `.only` statements are still in the tests
-      - run: "! git grep '\\.only' end-to-end-tests**/*spec* src**/*spec*"
+      - run: "! git grep '\\.only' $(find . -name '*spec*')"
```

Comment:
test failing.    i think we have to check for paren .  .only\(

Reply:
i just did an explicit ignore for those lines in oql parser with a `grep -v`